modules/client-configuration/pages/registration-cli.adoc

```diff
@@ -80,6 +80,11 @@ Salt ([package]``salt-minion``)::
 * [path]``/etc/salt/minion.d/NAME.conf``
 * systemd service file: [path]``salt-minion.service``
 
+[WARNING]
+====
+Bootstrapping this way will only work if you use the [package]``salt-minion`` being part of the client tools channels or of an official {suse} distributions.
+====
+
```

Comment:
I see this is included inside `.Procedure: Registering Clients with Salt Bundle Configuration File` but this also applies for the other procedure in this page (`.Procedure: Registering Clients with Salt Minion Configuration File`).

I would rather put this warning message before starting with the different procedures, so around line 34 of this page.

Reply:
I think in this case it would make sense to to have the warning even a little bit earlier.  Here is a new try now